test/resolve/connectionResolver.test.ts

```diff
@@ -168,6 +168,36 @@ describe('ConnectionResolver', () => {
       expect(metadataQueryStub.calledOnce).to.be.true;
     });
 
+    it('should resolve +100K components', async () => {
+      const promiseAllSpy = $$.SANDBOX.spy(Promise, 'all');
+      // @ts-expect-error spying on private method
+      const resolverSpy = $$.SANDBOX.spy(ConnectionResolver.prototype, 'sendBatchedQueries');
+
+      const metadataListStub = $$.SANDBOX.stub(connection.metadata, 'list');
+
+      const classesQty = 200_000;
```

Comment:
120K is enough to get this UT fail on my macbook but was passing on GHA runners, had to increase to 200K to get it fail 🤷🏼